indexer-events/src/main.rs

```diff
@@ -91,5 +92,22 @@ async fn handle_streamer_message(
 
     db_adapters::events::store_events(pool, &streamer_message, chain_id).await?;
 
+    match indexer_opts::update_meta(
+        &pool,
+        &indexer_id,
+        streamer_message.block.header.height,
+    )
+    .await
+    {
+        Ok(_) => {}
+        Err(err) => {
+            tracing::warn!(
+                target: crate::LOGGING_PREFIX,
+                "Failed to update meta for INDEXER ID {}\n{:#?}",
+                indexer_id,
+                err,
+            );
+        }
+    };
```

Comment:
I don't think we need to update this for each block.
Historical indexers can load up to 400 blocks per second. With this change, the speed will degrade dramatically
I like the idea to update meta each 100 blocks. 

Another suggestion: let's move logging inside `update_meta`? We will have this exact code everywhere

Reply:
We even already have the logging there, so it could be simplified to smth like
`let _ = update_meta().await;`